css/media.css

```diff
@@ -20,29 +20,36 @@
         margin-left: 25%; 
         height: auto;      
     }
-    #header, #navbar {
+     #navbar {
         display:flex;
-        flex-direction: column;       
+        flex-direction: column;
+        height: 100%;       
     } 
-    #navbar {
-        height: 100%;    
+    #header button {
+        color: #f9f92f;  
+        background-color:  #555653; 
+        width: 100%;
     }
-    #navbar li {
-        padding: 0;
-        display:block;
-        border-bottom: 1px solid #a9a923;                 
+    #header button i {
+        padding-left: 1em;
+    }
+    #navbar li {        
+        border-bottom: 1px solid #a9a923;                           
     }
     #navbar a {       
-        font-size: 12px;
+        font-size: 14px;
         padding-left: 3em;
-    } 
+    }    
     .grid-container p {
         padding: 0.5rem;
         margin: 0.25rem;
     }
     .skillsBox li {
         margin-left: 0.25rem;
-    }    
+    }  
+    #projects li:hover {
+        width: 50%;
+    }  
```

Comment:
Nitpick: When you set this width to 50%, on mobile devices, it causes the text to wrap and shift all of the content on the page down:
![shifting_Text](https://github.com/ofeterovskaya/fenrir-intro/assets/22607722/b0e77c30-1ffb-47c5-b4a3-61be45df57d8)

I'm not sure why this element needs its width to be set to 50% on hover, so I can't provide any recommendations on how to resolve it.

Reply:
Thank you Matt. I have a question. Do you think Shade effect makes h2 readable? Or I should to rid this off?

Reply:
In this case, having a dark font color against a dark background color is not ideal. Typically, you'll want the highest contrast available: dark text -> light background or light text -> dark background. There are some online tools that measure this "color contrast ratio". Depending on the font color and font size, you'll typically want a contrast ratio of 4.5-7. Therefore, I think it's fine to have the dark color on hover, but it should only be applied when the link's text changes from black to yellow (as shown in the second screenshot).

Reply:
Apologies, I misread your question. I've updated my answer above!